src/Model/Subscription.php

```diff
@@ -33,70 +36,67 @@ class Subscription extends Entity implements Arrayable
     public function __construct(?string $id = null)
     {
         parent::__construct($id);
+        $this->startDate = new DateTime('now', new DateTimeZone('UTC'));
```

Comment:
we do want this to be able to be null though... or we can run into issues signing up before 9am UTC. API has a behat that covers this weird case.

unless we add a check later that says if start date is today, remove it before sending to BT. We'll see - I haven't made it that far down the code yet.

Reply:
Yeah, I did make dates nullable - but this should probably be initialized to null. Good catch!